contracts/scripts/deploy.sh

```diff
@@ -126,7 +143,23 @@ ACCESS_CONTROL_PUBKEY=$(docker run --rm --entrypoint "/bin/sh" "${NODE_IMAGE}" -
 echo "access control contract address: " $ACCESS_CONTROL
 echo "access control contract public key (hex): " $ACCESS_CONTROL_PUBKEY
 
-# --- UPLOAD TOKEN CONTRACT CODE
+# --- UPLOAD TICKET TOKEN CONTRACT CODE
+
+cd "$CONTRACTS_PATH"/ticket_token
+# replace address placeholder with the on-chain address of the AccessControl contract
+link_bytecode ticket_token 4465614444656144446561444465614444656144446561444465614444656144 $ACCESS_CONTROL_PUBKEY
+# remove just in case
+rm target/ink/ticket_token.wasm
+# NOTE : here we go from hex to binary using a nodejs cli tool
```

Comment:
Not blocking: do we need this dependency? Would get rid of it, unless prohibitively expensive in terms of time.

Reply:
Sorry I don't quite understand: we are using it during deployment, if we get rid of it it stops working? Are you asking why we are using it?

Reply:
I guess my question is: "Why are we using a separate dep rather than keeping this inside the repo?" If this tool had been written before and was used for convenience, that's a perfectly acceptable answer. Otherwise, there's a separate dependency and another language to consider.